hazelcast-jet-all/src/main/java/com/hazelcast/jet/server/JetCommandLine.java

```diff
@@ -406,7 +406,7 @@ public void cluster(
             JetClientInstanceImpl client = (JetClientInstanceImpl) jet;
             HazelcastClientInstanceImpl hazelcastClient = client.getHazelcastClient();
             ClientClusterService clientClusterService = hazelcastClient.getClientClusterService();
-            Member masterMember = clientClusterService.getMember(clientClusterService.getMasterAddress());
+            Member masterMember = clientClusterService.getMember(clientClusterService.getMasterMember().getUuid());
```

Comment:
To get master member, we get the master member and get its uuid and then get the master member again ?
@jbartok